dev-client/src/components/content/HelpSection.tsx

```diff
@@ -0,0 +1,11 @@
+import {StyleSheet, View} from 'react-native';
+
+export const HelpSection = ({children}: React.PropsWithChildren) => {
+  return <View style={styles.help}>{children}</View>;
+};
+
+const styles = StyleSheet.create({
+  help: {
+    marginLeft: 6,
```

Comment:
This component handles the margin that was previously embedded in `HelpButton` / `InfoButton`. It seemed cleaner to have the buttons be stand-alone components that behave like their neighbors and have a container that handles spacing them around their surroundings.